packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -72,25 +68,12 @@ spec:
       coreDNSImage: {{ template "system_default_registry" . }}{{ .Values.istiocoredns.image.repository }}
       coreDNSPluginImage: {{ template "system_default_registry" . }}{{ .Values.istiocoredns.pluginImage.repository }}:{{ .Values.istiocoredns.pluginImage.tag }}
       coreDNSTag: {{ .Values.istiocoredns.image.tag }}
-    {{- if or .Values.policy.enabled .Values.telemetry.v1.enabled }}
-    mixer:
-      {{- if .Values.policy.enabled }}
-      policy:
-        image: {{ template "system_default_registry" . }}{{ .Values.policy.repository }}:{{ .Values.policy.tag }}
-      {{- end }}
-      {{- if .Values.telemetry.v1.enabled }}
-      telemetry:
-        image: {{ template "system_default_registry" . }}{{ .Values.telemetry.v1.repository }}:{{ .Values.telemetry.v1.tag }}
-      {{- end }}
-    {{- end }}
```

Comment:
This is because mixer is gone ?

Reply:
Yes, mixer is not supported in any form in 1.8. I did put a comment in the values.yaml during the 1.7 release that said it would be deprecated soon. This will be something that would need to go in release notes.